doc/modules/preprocessing.rst

```diff
@@ -69,34 +68,18 @@ Scaled data has zero mean and unit variance::
   >>> X_scaled.std(axis=0)
   array([1., 1., 1.])
 
-..    >>> print_options = np.set_printoptions(print_options)
-
-The ``preprocessing`` module further provides a utility class
-:class:`StandardScaler` that implements the ``Transformer`` API to compute
-the mean and standard deviation on a training set so as to be
+The class implements the ``Transformer`` API to compute the mean and standard
+deviation on a training set so as to be
 able to later reapply the same transformation on the testing set.
 This class is hence suitable for use in the early steps of a
 :class:`~sklearn.pipeline.Pipeline`::
 
-  >>> scaler = preprocessing.StandardScaler().fit(X_train)
-  >>> scaler
-  StandardScaler()
-
   >>> scaler.mean_
   array([1. ..., 0. ..., 0.33...])
 
   >>> scaler.scale_
   array([0.81..., 0.81..., 1.24...])
 
```

Comment:
I would move these lines right after fitting the `sklearn.preprocessing.StandardScaler` estimator (line 55).

Reply:
Ok